drivers/lte_link_control/lte_lc.c

```diff
@@ -129,6 +137,20 @@ static int w_lte_lc_init_and_connect(struct device *unused)
 		return err;
 	}
 	LOG_INF("PDP Context: %s", cgdcont);
+#endif
+#if defined(CONFIG_LTE_NETWORK_MODE_LTE)
+	err = at_cmd(at_socket_fd, lte, AT_CMD_SIZE(lte));
```

Comment:
Since these added two options are mutually exclusive, wouldn't it be easier to have here something like:
```
err = at_cmd(at_socket_fd, network_mode, AT_CMD_SIZE(network_mode));
```
that would be executed always, and provide the definition of `network_mode` conditionally, according to the selected option?

Reply:
Agreed, update pushed